test/utils/helpers.go

```diff
@@ -435,10 +437,20 @@ func SetAgentConfig(c client.Client, ns *corev1.Namespace, cloudProviders, antre
 		return err
 	}
 
+	if withWindows {
```

Comment:
Put this block above the previous error check, and remove the nested error check. We don't really care about `filepath.Abs("./hack/install-vm-agent-wrapper.sh")` errors if we want windows

Reply:
Done